src/constants/links.js

```diff
@@ -0,0 +1,6 @@
+const Links = {
+  CONTRACT_URL: 'https://goerli.voyager.online/contract/',
```

Comment:
Why voyager and not starkscan?

Reply:
I asked you yesterday, and you told me to leave it with voyager

Reply:
To leave what? you added it as new URLs

Reply:
https://github.com/starkware-libs/cairo-playground/blob/6ea320a859c0769c5035771c410896d4dbe6c11c/src/components/Features/Output/DeployOutput.js#L4
they were here. I just changed the location cause I needed `CONTRACT_URL` in `StarknetWalletButton`

Reply:
OK, so lets change it to starkscan please

Reply:
done